harvard-lts/etd-cron
# logging updates

**Adds Node logging based on some critical updates to the base template**
* * *

**GitHub Issue**: https://jira.huit.harvard.edu/browse/ETD-229

# How should this be tested?

Start up the local environment following the instructions in the [README](https://github.com/harvard-lts/etd-cron#local-development-environment-setup-instructions)
Verify that an audit and console file appear in the logs/ directory.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no
